app/models/info_request.rb

```diff
@@ -129,6 +129,7 @@ class InfoRequest < ActiveRecord::Base
   scope :embargoed, Prominence::EmbargoedQuery.new
   scope :not_embargoed, Prominence::NotEmbargoedQuery.new
   scope :embargo_expiring, Prominence::EmbargoExpiringQuery.new
+  scope :visible_to_requester, Prominence::VisibleToRequesterQuery.new
```

Comment:
this bugs me slightly as it only does what it says when applied to a user (I have no alternative suggestion)

Reply:
I'd read it as "all requests in a state that would allow the request owner to view them" – not quite as snappy for a scope name though 🤣

Reply:
I've added that description to the inline class documentation.